src/components/datepicker/readme.md

```diff
@@ -1,6 +1,16 @@
-# fw-datepicker
+# Datepicker (fw-datepicker)
+fw-datepicker displays calender with prepopulated month and year and enables to picking a date or date range. The calender displaying based mode attribute to select a single date or range of dates.
```

Comment:
Please modify this to "fw-datepicker displays an input box with a calendar that enables selecting a date or date range. The values preselected in the input box and calendar are based on the fw-datepicker attribute values."